shared/whats-new/container.tsx

```diff
@@ -20,89 +22,104 @@ type OwnProps = {
   onBack?: () => void
 }
 
-const mapStateToProps = (state: Container.TypedState) => ({
-  lastSeenVersion: state.config.whatsNewLastSeenVersion,
-})
-const mapDispatchToProps = (dispatch: Container.TypedDispatch) => ({
-  // Navigate primary/secondary button click
-  _onNavigate: ({
-    fromKey,
-    path,
-    replace,
-  }: {
-    fromKey?: string
-    path: Array<{props?: {}; selected: string}>
-    replace?: boolean
-  }) => {
-    dispatch(
-      RouteTreeGen.createNavigateAppend({
-        fromKey,
-        path,
-        replace,
-      })
-    )
-  },
+WhatsNew.navigationOptions = Container.isMobile
+  ? {
+      HeaderTitle: keybaseFM,
+      header: undefined,
+      title: keybaseFM,
+    }
+  : {}
 
-  _onNavigateExternal: (url: string) => openURL(url),
+const WhatsNewContainer = Container.namedConnect(
+  (state: Container.TypedState) => ({
+    lastSeenVersion: state.config.whatsNewLastSeenVersion,
+    updateAvailable: state.config.updateInfo.status === 'suggested',
+    updateMessage:
+      isLinux && state.config.updateInfo.status === 'suggested'
+        ? state.config.updateInfo?.suggested?.message
+        : '',
+  }),
+  (dispatch: Container.TypedDispatch) => ({
+    // Navigate primary/secondary button click
+    _onNavigate: ({
+      fromKey,
+      path,
+      replace,
+    }: {
+      fromKey?: string
+      path: Array<{props?: {}; selected: string}>
+      replace?: boolean
+    }) => {
+      dispatch(
+        RouteTreeGen.createNavigateAppend({
+          fromKey,
+          path,
+          replace,
+        })
+      )
+    },
 
-  _onUpdateLastSeenVersion: (lastSeenVersion: string) => {
-    const action = GregorGen.createUpdateCategory({
-      body: lastSeenVersion,
-      category: 'whatsNewLastSeenVersion',
-    })
-    dispatch(action)
-  },
-})
-const mergeProps = (
-  stateProps: ReturnType<typeof mapStateToProps>,
-  dispatchProps: ReturnType<typeof mapDispatchToProps>,
-  ownProps: OwnProps
-) => {
-  const seenVersions = getSeenVersions(stateProps.lastSeenVersion)
-  const newRelease = anyVersionsUnseen(stateProps.lastSeenVersion)
-  const onBack = () => {
-    if (newRelease) {
-      dispatchProps._onUpdateLastSeenVersion(currentVersion)
-    }
-    if (ownProps.onBack) {
-      ownProps.onBack()
+    _onNavigateExternal: (url: string) => openURL(url),
+
+    _onUpdateLastSeenVersion: (lastSeenVersion: string) => {
+      const action = GregorGen.createUpdateCategory({
+        body: lastSeenVersion,
+        category: 'whatsNewLastSeenVersion',
+      })
+      dispatch(action)
+    },
+
+    _onUpdateSnooze: () => {
+      // TODO @jacob PICNIC-684 - Add snoozing RPC
+      // TODO @jacob PICNIC-684 - Handle linux "never notify"
```

Comment:
TODO here for the next ticket

Reply:
what's the plan here? do the other prs before merging this into master i assume?

Reply:
@chrisnojima Yes that's the plan.